File: Java Barcode Scanner ME/app/src/main/java/de/dimedis/mobileentry/backend/response/GetVersionsResponseContent.java
package de.dimedis.mobileentry.backend.response;

import com.google.gson.annotations.SerializedName;

public class GetVersionsResponseContent extends BaseResponseContent {
    public static final String VERSIONS = "versions";
    public static final String STATUS = "status";
    public static final String __STUB__ = "__STUB__";


    @SerializedName(VERSIONS)
    private Versions mVersions;

    public Versions getVersions() {
        return mVersions;
    }

    @Override
    public String toString() {
        return super.toString() + "\n" + mVersions;
    }
}

File: Kotlin Migration/NTMobileEntry/app/src/main/java/de/dimedis/mobileentry/backend/response/GetVersionsResponseContent.kt
package de.dimedis.mobileentry.backend.response

import com.google.gson.annotations.SerializedName

class GetVersionsResponseContent : BaseResponseContent() {
    @SerializedName(VERSIONS)
    private val mVersions: Versions? = null

    val versions: Versions?
        get() = mVersions

//    fun getVersions(): Versions? {
//        return mVersions
//    }

    override fun toString(): String {
        return """
           ${super.toString()}
           $mVersions
           """.trimIndent()
    }

    companion object {
        const val VERSIONS = "versions"
        const val STATUS = "status"
        const val __STUB__ = "__STUB__"
    }
}